create_tables.sql

```diff
@@ -66,6 +66,7 @@ CREATE TABLE green_rental.control (
     controlDate DATETIME,
     controlLarge BOOL,
     fuelLevel INT,
+    mileage FLOAT,
```

Comment:
I think this is set through the vehicle relation, right?

Reply:
Apparently not according to our datamodel, but I wonder why we decided to it that way...

Reply:
I take that back :D